cwms-data-api/src/test/java/cwms/cda/api/location/kind/OutletControllerTestIT.java

```diff
@@ -105,7 +103,6 @@ public static void tearDown() throws Exception {
         tearDownProject();
     }
 
-    @Disabled("Disabled due to a DB issue.  See https://jira.hecdev.net/browse/CWDB-296")
```

Comment:
This should stay disabled until the integration tests point to a production database version with the bug fix. Currently the fix is only in our cda_staging branch of the schema.

Reply:
Disabled test again